implementation/contracts/deposit/DepositUtils.sol

```diff
@@ -68,26 +68,26 @@ library DepositUtils {
         mapping (bytes32 => uint256) approvedDigests;
     }
 
-    /// @notice         Gets the current block difficulty
-    /// @dev            Calls the light relay and gets the current block difficulty
-    /// @return         The difficulty
+    /// @notice         Gets the current block difficulty.
+    /// @dev            Calls the light relay and gets the current block.
```

Comment:
Why'd we lose the word `difficulty` here? This is the only fix I want to land before we merge.